### PR TITLE
Fix dcinside.com

### PR DESCRIPTION
**Pull request by ghost**
AdguardTeam/AdguardFilters#81513

### CODE REVIEW

File: SpywareFilter/sections/mobile_whitelist.txt [hunk 1]
```diff
@@ -2,6 +2,9 @@
 ! Exception rules that fix broken mobile apps and mobile websites
 !
 !
+! https://github.com/AdguardTeam/AdguardFilters/issues/81513
+@@||google-analytics.com^$script,domain=m.dcinside.com|mupload.dcinside.com
```
Comment (Sergey-Lyapin) [inline SpywareFilter/sections/mobile_whitelist.txt:6]: What about to make this rule more narrow? For example these 2 rules works good on my device, please check:
```
@@||google-analytics.com/analytics.js^$script,domain=m.dcinside.com|mupload.dcinside.com
@@||ssl.google-analytics.com/ga.js^$script,domain=m.dcinside.com|mupload.dcinside.com
```

Comment (piquark6046) [reply SpywareFilter/sections/mobile_whitelist.txt]: https://github.com/AdguardTeam/AdguardFilters/pull/81571/commits/754569f133cf9f63ab7ea5870238177620823d68

Comment (Sergey-Lyapin) [reply SpywareFilter/sections/mobile_whitelist.txt]: @piquark6046
Did you check them?

Comment (piquark6046) [reply SpywareFilter/sections/mobile_whitelist.txt]: @Sergey-Lyapin 
Yes. On AG browser extension, they work.

Comment (Sergey-Lyapin) [reply SpywareFilter/sections/mobile_whitelist.txt]: Ok, thank you

**Pull request merged**